deploy/scripts/teams/team.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Rename to match the purpose, eg. gen_team.sh or something like that.
Also add better description on how to use it and what it's good for.